src/city/view.c

```diff
@@ -220,13 +220,14 @@ void city_view_get_selected_tile_pixels(int *x_pixels, int *y_pixels)
     *y_pixels = data.selected_tile.y_pixels;
 }
 
-int city_view_pixels_to_grid_offset(int x_pixels, int y_pixels)
+map_point city_view_pixels_to_map_point(int x_pixels, int y_pixels)
```

Comment:
> I prefer passing return structs as out parameter. Returning it copies the entire struct onto the stack, which may be fine for small things like this map_point, but I'd rather have one approach.

If that's the case, I'm guessing I should pass `const touch *` instead of the `touch` structure in touch-related functions, right?